app/scripts/background.js

```diff
@@ -614,31 +616,21 @@ export async function loadStateFromPersistence() {
 
   // read from disk
   // first from preferred, async API:
-  versionedData =
+  const preMigrationVersionedData =
     (await localStore.get()) || migrator.generateInitialState(firstTimeState);
 
-  // check if somehow state is empty
-  // this should never happen but new error reporting suggests that it has
-  // for a small number of users
-  // https://github.com/metamask/metamask-extension/issues/3919
-  if (versionedData && !versionedData.data) {
-    // unable to recover, clear state
-    versionedData = migrator.generateInitialState(firstTimeState);
-    sentry.captureMessage('MetaMask - Empty vault found - unable to recover');
```

Comment:
Makes sense to delete this. I'm not seeing any references to this error in Sentry, and the linked issue suggests this hasn't occurred for a long time.